sdk/iot/hub/readme.md

```diff
@@ -0,0 +1,43 @@
+# Azure IoT Clients for Embedded C: Azure IoT Hub Client
+
+Official Embedded C client for Azure IoT Hub.
+
+## Getting started
+
+- **Docs**: If you're new to the Azure SDK for Embedded C take a look at the [State Machine diagram](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/iot/doc/mqtt_state_machine.md#device-provisioning-service) that explains the high-level architecture, SDK components, and a clear view of SDK x Application responsibilities. 
+- **Samples**: [This page](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot/hub/samples) explains samples for the Azure SDK for Embedded C and how to use them.
+
+## Need help?
+
+* File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-c/issues/new/choose).
+* Check [previous questions](https://stackoverflow.com/questions/tagged/azure+c) or ask new ones on StackOverflow using
+  the `azure` and `c` tags.
+
+## Contributing
```

Comment:
Contributing and license should refer to the repo contributing/license

```
## Contributing

If you'd like to contribute to this library, please read the [contributing guide][azure_sdk_for_c_contributing] to learn more about how to build and test the code.

### License

Azure SDK for Embedded C is licensed under the [MIT][azure_sdk_for_c_license] license.

<!-- LINKS -->
[azure_sdk_for_c_contributing]: https://github.com/Azure/azure-sdk-for-c/blob/master/CONTRIBUTING.md
[azure_sdk_for_c_license]: https://github.com/Azure/azure-sdk-for-c/blob/master/LICENSE
```

Reply:
done!